src/components/TodoItem.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable react/sort-comp */
+import React, { Component } from 'react';
+import { FaTrash } from 'react-icons/fa';
+import PropTypes from 'prop-types';
+import styles from './TodoItem.module.css';
+
+export default class TodoItem extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      editing: false,
+    };
+  }
+
+  handleEditing = () => {
+    this.setState({
+      editing: true,
+    });
+  };
+
+  // editTodo
+  handleUpdatedDone = (event) => {
+    if (event.key === 'Enter') {
+      this.setState({ editing: false });
+    }
+  };
+
+  componentWillUnmount() {
+    console.log('Cleaning up...');
```

Comment:
_[OPTIONAL]_ Although I was still able to review your project, this console.log compiled an **Unexpected console statement  "no-console"** kindly revisit it to resolve the  "no-console" statement